scu-de/src/fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2/Helpers/TaskHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2.Helpers
+{
+    public static class TaskHelper
+    {
+        public static async void ExecuteInBackgroundThread(this Task task, Action<Exception> onException = null)
```

Comment:
I don't think this method is used anywhere in the swissbitcloudv2

Reply:
doch wird cachedExport